### ISSUE TITLE
docs or notebook: better explain learning rate schedulers

### ISSUE THREAD

**Issue by benjamin-work** [skorch-dev/skorch#275]
skorch has pretty good support for learning rate schedulers but unfortunately, they are not well documented yet. I would like to see either a dedicated section in the docs or in a notebook that illustrates their usage. With the new addition of simulating the learning rate (see #269), it should be easy to include some plots.

**Comment by ParagEkbote** [comment]
Hi, could you please assign this issue to me? I have some experience in using optimizers such as  Adam,AdamW for finetuning using LoRA and would love to contribute a dedicated notebook in the docs. 👍

**Comment by BenjaminBossan** [comment]
@ParagEkbote Done, thanks.
